### PR TITLE
Check for Simple objects

### PR DESCRIPTION
**Pull request by rightisleft**
Looks like double wasn't added to the list - this enables the 'parser simple' to pass using doubles.

Also check for simply types instead of doing recursive lookups on all objects. This will prevent going down the tree to infinity for boxed objects.

### CODE REVIEW

File: lib/src/core_transformers.dart [hunk 1]
```diff
@@ -9,6 +9,7 @@ final Symbol _QN_BOOL = reflectClass(bool).qualifiedName;
 final Symbol _QN_LIST = reflectClass(List).qualifiedName;
 final Symbol _QN_MAP = reflectClass(Map).qualifiedName;
 final Symbol _QN_OBJECT = reflectClass(Object).qualifiedName;
+final Symbol _QN_DOUBLE = reflectClass(double).qualifiedName;
```
Comment (eredo) [inline lib/src/core_transformers.dart]: I cannot find any usage of this variable. Am I missing something?

**Comment by eredo** [comment]
I did a merge. However I'm currently seeing some issues with running the tests in compiled mode. Not quiet sure if it's 1.13-dev related or something I just haven't checked so far. I'll do some more investigation and then complete the merge.

**Comment by rightisleft** [comment]
Hi - looking are core_transformers.dart i see your point. I think i initially tried to modify the _convert* functionality.

 However, i was able to block the infinite loop by checking _isSimpleType inside the serialize* functions. I also added int and double to _isSimpleType.

**Comment by rightisleft** [comment]
If you look at the output in https://github.com/eredo/dartson/issues/27 - you can see that the serialize function is trying to parse all of the helper methods for the numeric classes, isFinite, isNan, etc. etc

**Comment by eredo** [comment]
Closed due to 1.0.0-alpha

**Pull request closed without merging**